src/pages/QuizPage/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react";
+import { InstructionsComponent, QuizCard } from "../../components";
+import { Heading } from "../../components/CategoryCard/styles";
+import { useQuizData } from "../../context/quizContext";
+import { PageWrapper } from "./styles";
+import axios from "axios";
+import { SET__QUESTIONS } from "../../constants";
+import { CancelButton } from "../../components/Modal/AddName/styles";
+import { useHistory } from "react-router";
+
+const { REACT_APP_BACKEND_URL } = process.env;
+
+function QuizPage() {
+	const { dispatch, state } = useQuizData();
+	const [quizQuestion, setQuizQuestions] = useState([]);
+	const [questionNumber, setQuestionNumber] = useState(0);
+	const history = useHistory();
+
+	useEffect(() => {
+		(async () => {
+			const response = await axios.get(
+				`${REACT_APP_BACKEND_URL}/quiz?category=${state.modal.data}`
+			);
+			setQuizQuestions(response.data.questions);
```

Comment:
Response can be destructed here `const {data:{questions}}`

Reply:
done